backend/app/services/chat/storage/task_manager.py

```
@@ -247,6 +251,20 @@ def create_new_task(
     if not task_kinds_service.validate_task_id(db, new_task_id):
         raise HTTPException(status_code=500, detail="Failed to create task ID")
 
+    if params.project_id:
+        project_exists = (
+            db.query(Project.id)
+            .filter(
+                Project.id == params.project_id,
+                Project.user_id == user.id,
+                Project.client_origin == params.client_origin,
+                Project.is_active == True,
```

Ruff (0.15.14)

[error] Avoid equality comparisons to `True`; use `Project.is_active:` for truth checks

Replace with `Project.is_active`

(E712)